includes/cc-picker.js

```diff
@@ -0,0 +1,3 @@
+jQuery(document).ready(function($){
+    $('.my-color-field').wpColorPicker();
```

Comment:
You're using spaces instead of tabs again.

Reply:
its become my most used vscode command